src/redux/books/Books.js

```diff
@@ -1,13 +0,0 @@
-import React from 'react';
-
-const Books = () => (
-
-  <form>
-    <input type="text" placeholder="Book Title" />
-    <input type="text" placeholder="Book Author" />
-    <button type="button">Add Book</button>
-  </form>
```

Comment:
- [ ] It looks like you delete this <form> inside redux, but I'm afraid I couldn't find it in the application. Please add it. 🐶